php-config/Slate/UI/Tools.config.d/cbl.php

```diff
@@ -29,8 +29,10 @@
 
 if ($GLOBALS['Session']->Person && !empty($GLOBALS['Session']->Person->Wards)) {
     foreach($GLOBALS['Session']->Person->Wards as $Ward) {
-        $cblTools[$Ward->FirstNamePossessive . ' Competency Dashboard'] = '/cbl/dashboards/demonstrations/student?student='.$Ward->Username;
-        $cblTools[$Ward->FirstNamePossessive . ' Task Dashboard'] = '/cbl/dashboards/tasks/student?student='.$Ward->Username;
+        if ($Ward instanceof \Slate\People\Student && $Ward->AccountLevel != 'Disabled') {
```

Comment:
@nbey looks good but it's better style to have one or more "guard clauses" at the top of a block than to deeply nest the whole body https://sourcemaking.com/refactoring/replace-nested-conditional-with-guard-clauses

When you find yourself wrapping the whole body of a function/loop in conditions, invert the conditions and use them to return/continue at the top instead